containers/ecr-viewer/integration/extended_database_repo.test.ts

```diff
@@ -61,176 +18,335 @@ describe("extended_database_repo", () => {
     await dropExtended();
   });
 
-  afterEach(async () => {
-    await clearExtended();
-  });
-
   describe("ecr_data", () => {
+    const template = {
+      eICR_ID: "12345",
+      set_id: "12345",
+      fhir_reference_link: "http://example.com",
+      last_name: "Kenobi",
+      first_name: "Obi-Wan",
+      birth_date: new Date("2024-12-31T05:00:00.000Z"),
+      gender: "Based",
+      birth_sex: "Based",
+      gender_identity: "Based",
+      race: "Star Guy",
+      ethnicity: "Star Guy",
+      latitude: 0.0,
+      longitude: 0.0,
+      homelessness_status: "Homeless",
+      disabilities: "None",
+      tribal_affiliation: "None",
+      tribal_enrollment_status: "None",
+      current_job_title: "Jedi Master",
+      current_job_industry: "Jedi Order",
+      usual_occupation: "Jedi Master",
+      usual_industry: "Jedi Order",
+      preferred_language: "Galactic Basic",
+      pregnancy_status: "Not Pregnant",
+      rr_id: "12345",
+      processing_status: "Processed",
+      eicr_version_number: "1.0",
+      authoring_date: new Date("2024-12-31T05:00:00.000Z"),
+      authoring_provider: "Dr. Droid",
+      provider_id: "12345",
+      facility_id: "12345",
+      facility_name: "Jedi Temple",
+      encounter_type: "Checkup",
+      encounter_start_date: new Date("2024-12-31T05:00:00.000Z"),
+      encounter_end_date: new Date("2024-12-31T05:00:00.000Z"),
+      reason_for_visit: "Checkup",
+      active_problems: ["Dead"],
+      date_created: new Date("2025-01-01"),
+    };
+    beforeEach(async () => {
+      await extended_database_repo.createExtendedEcr(template);
+    });
+
+    afterEach(async () => {
+      await clearExtended();
+    });
+
     it("should find an ECR with a given eICR_ID", async () => {
-      await extended_database_repo.findExtendedEcrById("12345");
+      const actual = await extended_database_repo.findExtendedEcrById("12345");
+      expect(actual).toEqual({
+        ...template,
+        active_problems: expect.stringContaining("Dead"),
+      });
     });
 
     it("should find all people named General", async () => {
-      await extended_database_repo.findExtendedEcr({ first_name: "General" });
+      const actual = await extended_database_repo.findExtendedEcr({
+        first_name: "Obi-Wan",
+      });
+      expect(actual[0]).toEqual({
```

Comment:
for this test set and others, they would be strengthened by creating multiple records (two should be fine) so we ensure specificity and can test length returned. I think they could mostly be a copy, just change the name and id